data/config-schema/src/main/resources/net/corda/schema/configuration/p2p.linkManager/1.0/corda.p2p.linkManager.json

```diff
@@ -89,10 +89,16 @@
       "default": 10000,
       "minimum": 500
     },
-    "sessionsPerPeer": {
-      "description": "The number of actively maintained sessions between two peers.",
+    "sessionsPerPeerForMgm": {
```

Comment:
Would we have backward compatibility issues?

Should/can we keep the `sessionsPerPeer` as a deprecated configuration?

Reply:
Also, maybe it can be less flat? Something like:
```
"sessionsPerPeer": {
  "forMgm": 1,
  "forMember": 2
}
```

Reply:
Good point!